styleguide.config.js

```diff
@@ -34,7 +34,6 @@ module.exports = {
             'src/elements/table/body.jsx',
             'src/elements/table/row.jsx',
             'src/elements/table/cell.jsx',
-            'src/elements/table/body.jsx',
```

Comment:
Why is this changed?

Reply:
it's duplicated. The body is declared three lines up. This duplicate value causes warning about the same key. (not sure why really)